src/components/CoSign/Cosign.js

```diff
@@ -0,0 +1,106 @@
+import React, {useContext} from 'react';
+import {Route, Routes, useNavigate} from 'react-router-dom';
+import {useImmerReducer} from 'use-immer';
+
+import {destroy} from 'api';
+import ErrorBoundary from 'components/ErrorBoundary';
+import {CosignSummary} from 'components/Summary';
+
+import {ConfigContext} from '../../Context';
+import CosignDone from './CosignDone';
+
+const initialState = {
+  submission: null,
+  privacyInfo: {
+    requiresPrivacyConsent: true,
+    privacyLabel: '',
+    policyAccepted: false,
+  },
+  summaryData: [],
+  reportUrl: '',
+  cosignedSubmission: null,
+};
+
+const reducer = (draft, action) => {
+  switch (action.type) {
+    case 'SUBMISSION_LOADED': {
+      draft.submission = action.payload;
+      break;
+    }
+    case 'LOADED_PRIVACY_INFO': {
+      draft.privacyInfo = {...draft.privacyInfo, ...action.payload};
```

Comment:
does this contain the full thing or only parts? If it's the full thing, you should be able to do `draft.privacyInfo = action.payload`.

Re-using a part from `draft` with immer reducers is a bit of a code smell because you should be able to just make the necessary mutations instead of needing to repeat the 'base' state.